Create vApp VM Security Tag Tests (#1046)
vmware/terraform-provider-vcd: website/docs/r/vapp_vm.html.markdown

  using `vcd_org_vdc.vm_placement_policy_ids` (and optionally `vcd_org_vdc.default_compute_policy_id` to make it default).
  In this case, if the placement policy is not set, it will pick the VDC default on creation. It must be set explicitly
  if one wants to update it to another policy (the VM requires at least one Compute Policy), and needs to be set to `""` to be removed.
* `security_tags` - (Optional; *v3.9+*) Set of security tags to be managed by the `vcd_vapp_vm` resource.

>   To remove `security_tags` you must set `security_tags = []` and do not remove the attribute. Removing the attribute will cause that the tags remain unchanged and just stop being managed by this resource.

Reviewer: ```suggestion
  To remove `security_tags` you must set `security_tags = []` and do not remove the attribute. Removing the attribute will cause the tags to remain unchanged and just stop being managed by this resource.
```

Author: Updated

>   This is to keep backwards compatibility with existing security tags that were created by the `vcd_security_tags` resource.

Reviewer: ```suggestion
  This is to keep compatibility with existing security tags that were created by the `vcd_security_tags` resource.
```
It is not **backwards** compatibility. It's a separate functionality.
Alternatively:
```suggestion
  This is to be consistent with existing security tags that were created by the `vcd_security_tags` resource.
```

Author: Updated to the alternative

> ~> **Note:** Either `security_tags` attribute or [`vcd_security_tag`](/providers/vmware/vcd/latest/docs/resources/security_tag) resource

Reviewer: ```suggestion
~> **Note:** Only one of `security_tags` attribute or [`vcd_security_tag`](/providers/vmware/vcd/latest/docs/resources/security_tag) resource
```

Author: Updated

>   should be used as they would cause a behavioral conflict with each other.

Reviewer: ```suggestion
  should be used. Using both would cause a behavioral conflict.
```

Author: Updated
